### PR TITLE
Fix thebell.io annoyances

### PR DESCRIPTION
**Pull request by komachi**
Links to test:

[https://thebell.io/otskok-posle-koshmara-virus-protiv-lekarstv-i-svyazi-chemu-uchitsya-u-azii-i-kak-pravilno-myt-ruki/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https://thebell.io/otskok-posle-koshmara-virus-protiv-lekarstv-i-svyazi-chemu-uchitsya-u-azii-i-kak-pravilno-myt-ruki/)
[https://thebell.io/sobyanin-vvel-v-moskve-svobodnoe-poseshhenie-shkol/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https://thebell.io/otskok-posle-koshmara-virus-protiv-lekarstv-i-svyazi-chemu-uchitsya-u-azii-i-kak-pravilno-myt-ruki/)

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1542,13 +1542,13 @@ petri.com#?#.content_widget:has(> div.widget_content > div.hs-form-insert)
 fort-russ.com#?#.single-container > div.better-studio-shortcode:has(> div.bs-push-noti-inner)
 gatevidyalay.com#?#aside > section.widget-sidebar:has(> div.textwidget > section > form[id^="subscribe-"])
 it-actual.ru##.widget-area > aside.widget > div[class] form[method="post"]
-thebell.io#?#.sidebar-item:has(> .connect)
-thebell.io#?#.sidebar-item:has(> div[id*="ad-sidebar-"])
-thebell.io#?#.sidebar-item:has(> div:contains(weekly newsletter))
+thebell.io##.sidebar
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt]: too wide rule.

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -1542,13 +1542,13 @@ petri.com#?#.content_widget:has(> div.widget_content > div.hs-form-insert)
...
 thebell.io##.follow > .follow__title
-thebell.io##.follow > .follow-form > .follow-form-cell >.follow-form-line
-thebell.io##.follow > .follow-form > .follow-form-cell >.follow__example
-thebell.io#?#.follow > .follow-form > .follow-form-cell:has(> .button)
+thebell.io##.follow > .follow__subtitle
+thebell.io##.follow > .follow-form
+thebell.io##.appbox
+thebell.io##.subs-only__overlay
+thebell.io##article.subs-only:style(height: auto !important; overflow: visible !important;)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt]: it seems these rules are not required.

File: AnnoyancesFilter/sections/annoyances.txt [hunk 3]
```diff
@@ -1542,13 +1542,13 @@ petri.com#?#.content_widget:has(> div.widget_content > div.hs-form-insert)
 fort-russ.com#?#.single-container > div.better-studio-shortcode:has(> div.bs-push-noti-inner)
 gatevidyalay.com#?#aside > section.widget-sidebar:has(> div.textwidget > section > form[id^="subscribe-"])
 it-actual.ru##.widget-area > aside.widget > div[class] form[method="post"]
-thebell.io#?#.sidebar-item:has(> .connect)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt]: 1545-1547 shoud not be removed

File: AnnoyancesFilter/sections/annoyances.txt [hunk 4]
```diff
@@ -1542,13 +1542,13 @@ petri.com#?#.content_widget:has(> div.widget_content > div.hs-form-insert)
...
-thebell.io#?#.sidebar-item:has(> div[id*="ad-sidebar-"])
-thebell.io#?#.sidebar-item:has(> div:contains(weekly newsletter))
+thebell.io##.sidebar
 thebell.io##.follow > .follow__title
-thebell.io##.follow > .follow-form > .follow-form-cell >.follow-form-line
-thebell.io##.follow > .follow-form > .follow-form-cell >.follow__example
-thebell.io#?#.follow > .follow-form > .follow-form-cell:has(> .button)
+thebell.io##.follow > .follow__subtitle
+thebell.io##.follow > .follow-form
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt]:
```
thebell.io##.follow > .follow__subtitle
thebell.io##.follow > .follow-form
```
what about `thebell.io##whatabout`?:) This rule will hide 
image

**Comment by adranger** [comment]
@komachi please don't post direct links to websites on this repository.  When people click on them, site owners can see what page they came from and then circumvent your filters.

You can add `[](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=)` to your [Saved replies](https://github.com/settings/replies) to make it easier to post HTTP referer-free links on Github.

File: AnnoyancesFilter/sections/annoyances.txt (hunk 2 above)
Comment (komachi) [reply AnnoyancesFilter/sections/annoyances.txt]: It's required for subs-only articles (it's not a paywall, it's a free subscription, so no legal problems here). Check out first link in issue.

File: AnnoyancesFilter/sections/annoyances.txt (hunk 4 above)
Comment (komachi) [reply AnnoyancesFilter/sections/annoyances.txt]: It's too wide, we need to keep whatabout block, it's a tl;dr version of an article.

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (komachi) [reply AnnoyancesFilter/sections/annoyances.txt]: On article page, sidebar contains only annoyances, we should remove it completely.

On front page, it also contain some annoyances and also some can-be-useful info. I prefer variant without sidebar completely, but we can probably hide it only on article page if this is important.

thebell_without
thebell_with

Comment (Alex-302) [reply AnnoyancesFilter/sections/annoyances.txt]: ok

File: AnnoyancesFilter/sections/annoyances.txt (hunk 2 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/annoyances.txt]: ok. 
`thebell.io##article.subs-only:style(height: auto !important; overflow: visible !important;)` 
=> 
`thebell.io#$#article.subs-only { height: auto !important; overflow: visible !important; }` - AdGuard syntax

File: AnnoyancesFilter/sections/annoyances.txt (hunk 4 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/annoyances.txt]: ok

File: AnnoyancesFilter/sections/annoyances.txt (hunk 2 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/annoyances.txt]: ----
Added directly to filters. Please check for filters update after 1-2 hours.

**Pull request closed without merging**